docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/net/index.md

```diff
@@ -7,7 +7,111 @@ description: The simplest way to start capturing telemetry data is to implement
+and is called library instrumentation by opentelemetry community. It is partial
+auto-instrumentation as traces are generated automatically depending on settings
+provided by developer(s) during initialization phase.
+
+:::note
+The below description applies to v0.7.0.
+:::
+
+## How to instrument a .NET application automatically (Windows)
+
+OpenTelemetry .NET Automatic Instrumentation has to be installed via Powershell
+module, process require administrator permissions.
+
+```powershell
+# Download the module
+$module_url = "https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/v0.7.0/OpenTelemetry.DotNet.Auto.psm1"
+$download_path = Join-Path $env:temp "OpenTelemetry.DotNet.Auto.psm1"
+Invoke-WebRequest -Uri $module_url -OutFile $download_path -UseBasicParsing
+
+# Import the module to use its functions
+Import-Module $download_path
+
+# Install core files (online vs offline method)
+Install-OpenTelemetryCore
+```
+
+After installation, we can set service name and run application
+
+```powershell
+# Set up the instrumentation for the current PowerShell session.
+# Can be done via environment variable.
+Register-OpenTelemetryForCurrentSession -OTelServiceName "MyServiceDisplayName"
+```
+
+The final step is to configure the exporter endpoint, service and application name otherwise defaults will be used. In this example, the instrumentation will be configured by environment variables.
+
+* `OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf` - configures OTLP exporter to use OTLP HTTP protocol
+* `OTEL_EXPORTER_OTLP_ENDPOINT=http://collection-sumologic-otelcol.sumologic:55681` - environment variable configures the endpoint where telemetry data will be sent. The value of the variable points to the default Sumologic Kubernetes Collector.
+* `OTEL_SERVICE_NAME=SERVICE_NAME` - configure the service name. Ensure the string value represents its business logic, such as "FinanceServiceCall". This will appear as a tracing service name in Sumo Logic.
+* `OTEL_RESOURCE_ATTRIBUTES=application=APPLICATION_NAME` - configure the application name. This will appear as a tracing application name in Sumo Logic. Additional attributes can be added here as comma separated key=value pairs.
+
+More extensive description of all available options can be found [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/config.md)
+
+```powershell
+# Run your application with instrumentation
+.\MyNetApp.exe
+```
+
+
+### ASP.NET application on IIS
+
+ASP.NET application that runs on IIS requires two additional steps.
+
+#### Step 1. Setup IIS instrumentation
+
+This step requires to execute following command in powerhell.
+
+```powershell
+# Setup IIS instrumentation
+Register-OpenTelemetryForIIS
+```
+
+#### Step 2. Define opentelemetry modules loaded by IIS
+
+There are two options in order to achieve the goal.
+
+First, update application `web.config` with following section.
```

Comment:
```suggestion
First, update the `web.config` application with the following section.
```

Reply:
I had application's `web.config` in mind